modules/weboramaRtdProvider.js

```diff
@@ -1022,9 +1105,12 @@ function getDataFromLocalStorage(weboDataConf, cacheGet, cacheSet, defaultLocalS
         const profile = data[targetingSection];
         const valid = isValidProfile(profile);
         if (!valid) {
-          logWarn(`found invalid ${source} profile on local storage key ${localStorageProfileKey}, section ${targetingSection}`);
+          logger.logMessage(
+            `WARNING: found invalid ${source} profile on local storage key ${localStorageProfileKey}, section ${targetingSection}: `,
+            profile
+          );
 
-          return;
+          return [defaultProfile, true];
```

Comment:
also, when it considers the profile is not valid, the empty return had a negative impact in the rest of the code, now we make sure that we use the default profile and we log correctly (not using warning, but the standard log message, to be possible hide the message via configuration)